### ISSUE TITLE
Support for oracle solaris on SPARC 64bit processor

### ISSUE THREAD

**Issue by badre429** [dotnet/coreclr#8895]
is there any plane to support Oracle Solaris on SPARC 64bit processor

**Comment by surejam** [comment]
Yes

**Comment by surejam** [comment]
Yes, it already is.

**Comment by mikedn** [comment]
> Yes, it already is.

Nope. There's no mention of SPAR 64bit anywhere in the [roadmap](https://github.com/dotnet/core/blob/master/roadmap.md) or anywhere else.

**Comment by surejam** [comment]
I'm sorry, I misunderstood the question.

**Comment by jkotas** [comment]
If you would like to run .NET Core on SPARC processors, the closest thing that we is CppCodeGen option in https://github.com/dotnet/corert project.

**Comment by gkhanna79** [comment]
CC @richlander

**Comment by richlander** [comment]
SPARC processors are not on our roadmap. We're targeting x86/64 and ARM32/64. We have received very few requests for any other chips other than a few "would be nice" style requests. As always, we're happy to help others port the product to other platforms.

**Comment by mfe-** [comment]
With .NET 5 mono gets somehow merged into coreclr (if I understood correctly). Since mono supports sparc ( https://www.mono-project.com/docs/about-mono/supported-platforms/sparc/ ) any plans for .NET 5?
